app/models/user.rb

```diff
@@ -0,0 +1,35 @@
+class User < ActiveRecord::Base
+  belongs_to :role
```

Comment:
If you do `validates :role, presence: true` it'll check that the role_id is present and maps to a role